AutoCollection/ClientRequests.ts

```diff
@@ -1,15 +1,18 @@
 ///<reference path="..\Declarations\node\node.d.ts" />
 
 import http = require("http");
+import https = require("http");
```

Comment:
No, that's a mistake. I'm not sure why I didn't catch it in testing; the https module is only used here on node versions < 0.12, but I specifically tested for that. I'll investigate.